src/matchers/toBeAccessible/index.ts

```diff
@@ -15,6 +15,7 @@ const summarize = (violations: Result[]) =>
 
 interface MatcherOptions extends RunOptions {
   timeout?: number
+  reportFileName?: string
```

Comment:
Let's simplify this to just `filename`.

```suggestion
  filename?: string
```